### PR TITLE
Add pytz requirement

### PR DESCRIPTION
**Pull request by synapticarbors**
Fixes https://github.com/conda-forge/django-feedstock/issues/26

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,14 +9,14 @@ package:
...
-  sha256: b6f3b864944276b4fd1d099952112696558f78b77b39188ac92b6c5e80152c30
+  sha256: {{ sha256 }}
   patches:
     # Hard-code gdal and geos' paths to the corresponding conda packages.
     - libgdal.patch
     - libgeos.patch
 
 build:
-  number: 0
+  number: {{ build }}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Please use jinja only for the version number.

Comment (synapticarbors) [inline recipe/meta.yaml]: I've seen jinja templates used for a variety of fields that change between updates. Having the `build` as a template, I believe, makes it more prominent and less likely to be overlooked during subsequent updates.

**Comment by ocefpaf** [comment]
I don't use itIt breaks my auto-update script and makes the recipe harder
to read.

On Jun 2, 2017 9:05 AM, "Joshua Adelman" <notifications@github.com> wrote:

*@synapticarbors* commented on this pull request.
------------------------------

In recipe/meta.yaml
<https://github.com/conda-forge/django-feedstock/pull/27#discussion_r119840485>
:

>    patches:
     # Hard-code gdal and geos' paths to the corresponding conda packages.
     - libgdal.patch
     - libgeos.patch

 build:
-  number: 0
+  number: {{ build }}

I've seen jinja templates used for a variety of fields that change between
updates. Having the build as a template, I believe, makes it more prominent
and less likely to be overlooked during subsequent updates.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/conda-forge/django-feedstock/pull/27#discussion_r119840485>,
or mute the thread

**Comment by synapticarbors** [comment]
@ocefpaf  @kwilcox Any idea why this is failing on py27 for windows:

https://ci.appveyor.com/project/conda-forge/django-feedstock/build/1.0.63/job/ay9ocv214pou1v70#L12350

**Comment by ocefpaf** [comment]
I'm only on a mobile phone today. can't see the log properly. can you paste
the error here?

**Comment by synapticarbors** [comment]
```
import: u'django.contrib.gis.db.backends.spatialite'
import: u'django.contrib.gis.forms'
Traceback (most recent call last):
  File "C:\bld\django_1496401931226\test_tmp\run_test.py", line 293, in <module>
    import django.contrib.gis.forms
  File "C:\bld\django_1496401931226\_t_env\lib\site-packages\django\contrib\gis\forms\__init__.py", line 3, in <module>
    from .fields import (  # NOQA
  File "C:\bld\django_1496401931226\_t_env\lib\site-packages\django\contrib\gis\forms\fields.py", line 4, in <module>
    from django.contrib.gis.geos import GEOSException, GEOSGeometry
  File "C:\bld\django_1496401931226\_t_env\lib\site-packages\django\contrib\gis\geos\__init__.py", line 5, in <module>
    from .collections import (  # NOQA
  File "C:\bld\django_1496401931226\_t_env\lib\site-packages\django\contrib\gis\geos\collections.py", line 11, in <module>
    from django.contrib.gis.geos.geometry import GEOSGeometry, LinearGeometryMixin
  File "C:\bld\django_1496401931226\_t_env\lib\site-packages\django\contrib\gis\geos\geometry.py", line 11, in <module>
    from django.contrib.gis import gdal
  File "C:\bld\django_1496401931226\_t_env\lib\site-packages\django\contrib\gis\gdal\__init__.py", line 43, in <module>
    from django.contrib.gis.gdal.driver import Driver  # NOQA
  File "C:\bld\django_1496401931226\_t_env\lib\site-packages\django\contrib\gis\gdal\driver.py", line 5, in <module>
    from django.contrib.gis.gdal.prototypes import ds as vcapi, raster as rcapi
  File "C:\bld\django_1496401931226\_t_env\lib\site-packages\django\contrib\gis\gdal\prototypes\ds.py", line 9, in <module>
    from django.contrib.gis.gdal.libgdal import GDAL_VERSION, lgdal
  File "C:\bld\django_1496401931226\_t_env\lib\site-packages\django\contrib\gis\gdal\libgdal.py", line 49, in <module>
    lgdal = CDLL(lib_path)
  File "C:\bld\django_1496401931226\_t_env\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
TypeError: LoadLibrary() argument 1 must be string, not unicode
(C:\bld\django_1496401931226\_t_env) (root) C:\bld\django_1496401931226\test_tmp>if errorlevel 1 exit 1 
TESTS FAILED: django-1.11-py27_1
Command exited with code 1
```

**Comment by ocefpaf** [comment]
Just drop the GIS tests on py27+win for now. probably not worth fixing it.

**Comment by ocefpaf** [comment]
Thanks @synapticarbors!

**Pull request merged**